src/main/java/org/opentripplanner/analyst/request/SampleGridRenderer.java

```diff
@@ -183,7 +188,7 @@ public WTWD closeSample(WTWD zUp, WTWD zDown, WTWD zRight, WTWD zLeft) {
                  * The computations below are approximation, but we are on the edge anyway and the
                  * current sample does not correspond to any computed value.
                  */
-                z.wTime = tMin + gridSizeMeters / v0;
+                z.wTime = tMin + gridSizeMeters / offRoadSpeed;
```

Comment:
This didn't merge correctly because these variables were renamed in functions that have been moved into a separate non-anonymous class. The changes look straightforward, I'll just apply them to WTWDAccumulativeMetric in master.